### ISSUE TITLE
MDX error

### ISSUE THREAD

**Issue by akshay-8d66** [Redocly/developer-portal-starter#178]
When using [e472582](https://github.com/Redocly/developer-portal-starter/commit/e472582d125d7cfd536dbab0085cc4e4acff9a4e) and going to a page with MDX directly: http://localhost:3000/developer-portal/mdx/, and an error is seen in Chrome developer  console -

```
Warning: Invalid DOM property `class`. Did you mean `className`?
    in div (created by MDXTag)
    in MDXTag (created by Context.Consumer)
    in Unknown (at mdx.mdx:43)
```

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_3u2j0)

**Comment by akshay-8d66** [comment]
Found on `1.1.0-beta.76` to `1.1.0-beta.79`. I didn't check before that.

**Comment by RomanHotsiy** [comment]
Hi @akshay-8d66,

Unfortunately we couldn't reproduce it. Tried on different machines. Could you provide some more details?

Thanks.

**Comment by RomanHotsiy** [comment]
@akshay-8d66 sorry for early notice. We were able to reproduce it now. We'll keep you posted.

**Comment by DmitryAnansky** [comment]
Hi @akshay-8d66,
This should be fixed in `v1.1.0-beta.80.patch.1`
Thanks.

**Comment by DmitryAnansky** [comment]
Resolved in 1.1.0-beta.80.patch.1
